src/js/item.ts

```diff
@@ -27,7 +29,11 @@ export function extractItems(
     order_id: string,
     order_date: string,
     order_detail_url: string,
+
+    // this is the only input essential to this function, the rest are just merged with the output hash
```

Comment:
Are you saying that we should remove these parameters?

Reply:
the same initial thought process applies - IF it's not needed for the category feature, it's a distraction and this conversation belongs in another pull request - I will gladly look at ways to remove dead code (there's probably a lot of it lying about that I've kitchen-sinked in)

Reply:
>Are you saying that we should remove these parameters?

Not necessarily, just documenting this for the future reader as this was confusing to me when reading the code.